src/web/app/components/error-report/error-report.component.ts

```diff
@@ -20,13 +21,19 @@ export class ErrorReportComponent implements OnInit {
   content: string = '';
   requestId: string = '';
   sendButtonEnabled: boolean = true;
+  errorReportEnabled: boolean = true;
   errorReportSubmitted: boolean = false;
+  readonly supportEmail: string = environment.supportEmail;
 
   constructor(private errorReportService: ErrorReportService,
               private ngbActiveModal: NgbActiveModal,
               private statusMessageService: StatusMessageService) {}
 
   ngOnInit(): void {
+    if (this.errorMessage === 'Missing CSRF token.') {
```

Comment:
Are they any other error messages as well that would be related to CSRF token issue? Such as `Invalid CSRF token.`

Reply:
Yep, `Invalid CSRF token.` is the only other one I could find, I'll update it to include that as well